state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -1618,6 +1622,14 @@ pub mod pallet {
 
 			let mut swap_output = CcmSwapOutput::default();
 
+			// Permill maxes out at 100% so this is safe.
+			let fee: u128 = Permill::from_parts(
```

Comment:
Pull into function? - Ideally we can combine these two whole deposit functions as I think you were referring to yesterday - but that can be done on `main`

Reply:
Yes the change is much smaller on main - this code path has already been consolidated with the other.